### PR TITLE
Hook cd in a simpler way

### PR DESCRIPTION
**Pull request by brycefisher**
@bobzoller @demands 

As requested by Max :-)

### CODE REVIEW

File: nvmish.sh [hunk 1]
```diff
@@ -75,17 +75,10 @@ function nvmish() {
   return 0
 }
 
-
 # This whole chpwd support business was lovingly borrowed from rvm.
 __dirname=$(brew --prefix nvmish)
```
Comment (bobzoller) [inline nvmish.sh]: this line can go too.

**Comment by brycefisher** [comment]
@bobzoller thanks for the review! What's the process to push out a new version?

**Comment by bobzoller** [comment]
just bump the version in the nvmish Formula file.

**Comment by demands** [comment]
Thanks @brycefisher!

**Pull request merged**